src/main/java/com/commercetools/sync/services/CustomObjectService.java

```diff
@@ -0,0 +1,74 @@
+package com.commercetools.sync.services;
+
+
+import com.commercetools.sync.commons.models.NonResolvedReferencesCustomObject;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customobjects.CustomObject;
+import io.sphere.sdk.customobjects.CustomObjectDraft;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.Optional;
+import java.util.concurrent.CompletionStage;
+
+public interface CustomObjectService {
+
+    /**
```

Comment:
Please adjust the Javadocs accordingly after addressing the review. Keep in mind, there are a lot of implementation detail that should not be in the Javadoc in the `interface` which does not know about the implementation (there could be multiple implementations)

Reply:
f3a06bbe350aa6fc0f780a15c05b455d580e7491